src/app/explore/explore.component.html

```diff
@@ -88,51 +88,65 @@
                        -->
-              <biggive-campaign-card
-                id="campaign-{{ campaign.id }}"
-                [banner]="campaign.imageUri | optimisedImage: 830 | async"
-                [campaignType]="campaign.isMatched ? 'Match Funded' : null"
-                [campaignTitle]="campaign.title"
-                [donateButtonUrl]="
-                  campaign.isRegularGiving ? '/regular-giving/' + campaign.id : '/donate/' + campaign.id
-                "
-                [donateButtonLabel]="campaign.isRegularGiving ? 'Donate monthly' : 'Donate now'"
-                [moreInfoButtonUrl]="'/campaign/' + campaign.id"
-                [organisationName]="campaign.charity.name"
-                [primaryFigureLabel]="
-                  campaign.isRegularGiving || campaign.parentUsesSharedFunds ? '' : 'Match Funds Remaining'
-                "
-                [primaryFigureAmount]="
-                  this.metaCampaign?.usesSharedFunds || campaign.isRegularGiving || campaign.parentUsesSharedFunds
-                    ? ' '
-                    : (campaign.matchFundsRemaining
-                      | currency: campaign.currencyCode : 'symbol' : currencyPipeDigitsInfo)
-                "
-                [progressBarCounter]="campaign.isRegularGiving ? null : getPercentageRaised(campaign)"
-                secondary-figure-label="Amount Raised"
-                [secondaryFigureAmount]="
-                  campaign.amountRaised | currency: campaign.currencyCode : 'symbol' : currencyPipeDigitsInfo
-                "
-                [isFutureCampaign]="isInFuture(campaign)"
-                [isPastCampaign]="isInPast(campaign)"
-                [datetime]="getRelevantDateAsStr(campaign)"
-              >
-              </biggive-campaign-card>
+              @if (campaign.parentUsesSharedFunds || campaign.isRegularGiving) {
+                <biggive-campaign-card
+                  id="campaign-{{ campaign.id }}"
+                  [banner]="campaign.imageUri | optimisedImage: 830 | async"
+                  [campaignType]="campaign.isMatched ? 'Match Funded' : null"
+                  [campaignTitle]="campaign.title"
+                  [donateButtonUrl]="
+                    campaign.isRegularGiving ? '/regular-giving/' + campaign.id : '/donate/' + campaign.id
+                  "
+                  [donateButtonLabel]="campaign.isRegularGiving ? 'Donate monthly' : 'Donate now'"
+                  [moreInfoButtonUrl]="'/campaign/' + campaign.id"
+                  [organisationName]="campaign.charity.name"
+                  [primaryFigureLabel]="'Amount Raised'"
+                  [primaryFigureAmount]="
+                    campaign.amountRaised | currency: campaign.currencyCode : 'symbol' : currencyPipeDigitsInfo
+                  "
+                  [progressBarCounter]="undefined"
+                  secondary-figure-label=""
+                  [secondaryFigureAmount]="' '"
```

Comment:
```suggestion
                  secondary-figure-amount=" "
```

Reply:
Not sure - In general yes the second form without the property binding is simpler and I'd prefer it, but I think in this case where almost everything else has to be property-bound it makes for easier reading when we use property binding on this too as I had done.